tests/test_nginx.py

```diff
@@ -1,15 +1,27 @@
 """This module contains the tests for the nginx container, the image with nginx pre-installed.
 """
-from bci_tester.data import NGINX_CONTAINER
+import requests
+from tenacity import retry
+from tenacity import stop_after_attempt
+from tenacity import wait_exponential
 
+from bci_tester.data import NGINX_CONTAINER
 
 CONTAINER_IMAGES = (NGINX_CONTAINER,)
 
 
-def test_nginx_welcome_page(auto_container, host):
+def test_nginx_welcome_page(auto_container):
     """test that the default welcome page is served by the container."""
     host_port = auto_container.forwarded_ports[0].host_port
 
-    assert "Welcome to nginx" in host.check_output(
-        f"curl -sf --retry 5 --retry-connrefused http://localhost:{host_port}/"
+    # Retry 5 times with quadratic backoff delay
```

Comment:
not exponential?

Reply:
Leftover when playing around with tenacity. Yes, it's exponential now.